ccd-definition/ComplexTypes/EmploymentTypeComplexTypeLRspec.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
this needs to be removed as no longer used.
To double check first before removing

Reply:
Removed